### ISSUE TITLE
HOOBS Certified Plugin not Replacing non Certified Version

### ISSUE THREAD

**Issue by askovi** [hoobs-org/HOOBS#296]
**Description**

Config options for Ring not loading. Steps to reproduce:
1. Install regular Ring plugin
2. Install HOOBS certified Ring plugin without removing the first one
3. Realize your mistake when they collide in logs
4. Uninstall non certified version. 
5. Try to load certified Ring configuration
![image](https://user-images.githubusercontent.com/3981552/70384719-7b4e5d00-1951-11ea-9465-802876865aea.png)

**Version**
3.1.1

**Comment by dgreif** [comment]
@askovi let me know if you need anything on my end to fix this.

I had another issue come up that I was hoping you can shed some light on.  A couple of my users have had weird config output from the HOOBS UI.  Specifically, boolean fields in the config have been saved to the config as strings instead of boolean values.  See https://github.com/dgreif/ring/issues/161#issuecomment-565343085 for reference.

**Comment by mkellsy** [comment]
@themick4u has this issue and was reported in issue #394 

Plugins array
```
"plugins": [
    "homebridge-tuya-web",
    "homebridge-magichome",
    "homebridge-wink3",
    "homebridge-dummy",
    "homebridge-camera-ffmpeg",
    "wink"
]
```

Notice there is both **homebridge-wink3** and **wink**

Output from `ls -la ~/.hoobs/node_modules`
```
hoobs@hoobs:~ $ ls -la ~/.hoobs/node_modules
total 1280
drwxr-xr-x 314 hoobs hoobs 12288 Dec 20 16:31 .
drwxr-xr-x   4 hoobs hoobs  4096 Jan  3 10:10 ..
drwxr-xr-x   3 hoobs hoobs  4096 Dec 10 20:22 homebridge-camera-ffmpeg
drwxr-xr-x   4 hoobs hoobs  4096 Dec  4 15:29 homebridge-dummy
drwxr-xr-x   3 hoobs hoobs  4096 Nov 19 20:52 homebridge-magichome
drwxr-xr-x   4 hoobs hoobs  4096 Nov 19 20:52 homebridge-tuya-web
drwxr-xr-x   7 hoobs hoobs  4096 Dec 20 16:28 @hoobs
```

> Truncated to show only plugins

And output from `ls -la ~/.hoobs/node_modules/@hoobs`
```
hoobs@hoobs:~ $ ls -la ~/.hoobs/node_modules/@hoobs
total 36
drwxr-xr-x   7 hoobs hoobs  4096 Dec 20 16:28 .
drwxr-xr-x 314 hoobs hoobs 12288 Dec 20 16:31 ..
drwxr-xr-x   4 hoobs hoobs  4096 Dec 20 16:28 wink
```

> Truncated to show only plugins

Notice that **homebridge-wink3** is not installed, so the certified version is getting installed but the **plugins** array is not getting updated.

**Comment by mkellsy** [comment]
This is fixed in 3.1.19.
